vite.config.js

```diff
@@ -7,7 +8,7 @@ export default defineConfig({
     plugins: [vue(), tailwindcss()],
     resolve: {
         alias: {
-            '@': path.resolve(__dirname, './src'),
+            '@': fileURLToPath(new URL('./src', import.meta.url)),
```

Comment:
I brought this over from the WW template, I thought it would fix the autocomplete when importing files, but thats gotta be another issue. Leaving this in for the future though.